community/kernel/src/main/java/org/neo4j/kernel/impl/api/OperationsFacade.java

```diff
@@ -1597,7 +1609,7 @@ private Object callFunction(
     {
         statement.assertOpen();
 
-        try ( KernelTransaction.Revertable ignore = tx.overrideWith( mode ) )
+        try ( KernelTransaction.Revertable ignore = tx.overrideWith( tx.securityContext().freeze( mode ) ) )
```

Comment:
It's just a way to associate the same subject with a new mode (within a SecurityContext). We'll change this to `withMode` to make it more clear.